src/modules/Bootloader.rb

```diff
@@ -150,7 +152,21 @@ def Read
       Progress.NextStage
       return false if testAbort
 
-      ::Bootloader::BootloaderFactory.current.read
+      begin
+        ::Bootloader::BootloaderFactory.current.read
+      rescue ::Bootloader::UnsupportedBootloader => e
+        ret = Yast::Report.AnyQuestion(_("Unsupported Bootloader"),
+          _("System uses unsupported bootloader '%s'. Propose to use supported bootloader?") %
```

Comment:
It would be "The system". But moreover, I think the phrasing is not crystal clear for the average user. Maybe something like: "Unsupported bootloader '%s' detected. Switch to a supported one?". Even better, since I assume we know the name of the proposed booloader: "Switch to '%s'?"